Resources/Textures/Objects/Tools/lockpick.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/94f27c7b25bbde39c1412e24fb32bf6470fcc394",
```

Comment:
I could be wrong, but didn't you make the sprite yourself? Give yourself credit and add your name here! Even if you just took a sprite and modified it slightly also mention that.

Reply:
GUH I FORGOR AHHHHHHHHHHHH